pkg/kubelet/kubelet_node_status.go

```diff
@@ -535,9 +534,11 @@ func (kl *Kubelet) syncNodeStatus() {
 // updateNodeStatus updates node status to master with retries if there is any
 // change or enough time passed from the last sync.
 func (kl *Kubelet) updateNodeStatus(ctx context.Context) error {
+	var lastErr error
 	klog.V(5).InfoS("Updating node status")
 	for i := 0; i < nodeStatusUpdateRetry; i++ {
-		if err := kl.tryUpdateNodeStatus(ctx, i); err != nil {
+		if err := kl.tryUpdateNodeStatus(ctx, lastErr); err != nil {
+			lastErr = err
```

Comment:
```suggestion
		if err := kl.tryUpdateNodeStatus(ctx, try); err != nil {
			if err != nil && apierrors.IsConflict(err) {
				try = true
			}
```
I think it's better to input bool type parameters instead of error...

Reply:
Done, tks